pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   grpc: ^2.1.3
   protobuf: ^1.0.1
   fixnum: ^0.10.11
+  googleapis_auth: ^0.2.11
```

Comment:
I don't think this is being used :-)

Reply:
On final comment: could you move the "closes #2" text from the subject to the description of your commit/pull request?